tools/ansible/tasks/logging/test/logging_test.go

```diff
@@ -37,7 +55,9 @@ func TestPluginsInstalled(t *testing.T) {
 		installedVersion, ok := installedPlugins[pluginName]
 		assert.Truef(t, ok, "Missing plugin %s", pluginName)
 		if pluginVersion != "" {
-			assert.Equalf(t, pluginVersion, installedVersion,
+			// Getting only the version in the format 1.5 instead of 1.5.6, because installation automatically
+			// fetches the latest subversion
```

Comment:
I did not understand what you mean by this? Perhaps an example? Where are you pinning the version?

Reply:
I meant here that even if we mention the installation version of fluentd as `1.5.1` while running the playbook, the installation automatically fetches the latest subversion, which in this case is `1.5.2`. So I have only added the check for the version, not the subversion (i.e. I'm only match `1.5` instead of `1.5.2`.) I can reword the comment so it explains it better.